dockstore-webservice/generated/src/main/resources/pom.xml

```diff
@@ -60,7 +60,7 @@
     <dependency>
       <groupId>io.dockstore</groupId>
       <artifactId>swagger-java-quay-client</artifactId>
-      <version>2.0.0</version>
+      <version>2.0.1-SNAPSHOT</version>
```

Comment:
This was likely in your local maven repo but not available in artifactory. 
You'll need to do a Maven release in the swagger-java-quay-client repo

Reply:
(the release will also make it not a snapshot)